ledger/ledger-api-integration-tests/BUILD.bazel

```diff
@@ -87,7 +87,12 @@ da_scala_test_suite(
     size = "large",
     # WARNING: srcs list has to be explicit here! It can not be added via deps, due to
     # test discovery mechanism in the test_suite macro working only on sources.
-    srcs = glob(["src/test/itsuite/**/*.scala"]),
+    srcs = glob(
+        include = ["src/test/itsuite/**/*.scala"],
+        # the excluded tests are run by the conformance test via the ledger-api-test-tool
+        # see //ledger/sandbox:conformance-test-*
+        exclude = ["src/test/itsuite/**/TransactionServiceIT.scala"],
```

Comment:
IMO, I feel like keeping it in the IT suite is still worth it as Bazel would do much better caching for those than to these tests via our tool. They are just better for tight iteration cycle.

Reply:
:+1:

Reply:
Having said that, your argument is still valid. And I also have concerns regarding the ergonomics of having to go through the ledger-api-tool to only run a specific integration tests.
For that we could add a bazel target that is not run in CI but still exposes the tests locally.